Add more GCMD keywords to exceedance limits dataset

IvanaH8 opened the pull request:
Adds 3 GCMD keywords related to 
- [acidification](https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/all/62ee81e7-9f5a-4af5-a086-f4c402c7d19d?gtm_keyword=ACID%20RAIN&gtm_scheme=Earth%20Science)
- [eutrophication](https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/all/53d36c39-3cb1-44db-9746-feee86cbe9d7?gtm_keyword=EUTROPHICATION&gtm_scheme=Earth%20Science) and 
- [hydrologic model](https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/all/2eb094d5-70bc-49fa-acb6-4ad07f4c7b08?gtm_keyword=HYDROLOGIC%20AND%20TERRESTRIAL%20WATER%20CYCLE%20MODELS&gtm_scheme=Earth%20Science%20Services).

The model keyword is maybe not necessary. There is also [Dynamic Vegetation/Ecosystem Models](https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/all/adfee6d2-ca00-4f02-a570-5ccf0850cb55?gtm_keyword=DYNAMIC%20VEGETATION/ECOSYSTEM%20MODELS&gtm_scheme=Earth%20Science%20Services) keyword that might be more suitable.

knl88 commented:
An inital draft of the dataset is available [here](https://thredds.t.niva.no/thredds/catalog/subcatalogs/models.html?dataset=samplewms-2017-2021) behind login until we can publish it:)

kariaust commented:
Keywords: Skip "hydrological model". I don't think any of the other model keywords fit either
The title is still wrong in the map
And I still find the projection a bit odd

knl88 commented:
Thanks @kariaust, for the projection the server has a default viewer that is mostly just for an initial look at the dataset, but the WMS server provides a bunch of [projections](https://thredds.t.niva.no/thredds/wms/datasets/exceedance-limits/fab-sswc-veg-exceedence/2017-2021_acdd_fab-sswc-veg-exceedence.nc?service=WMS&version=1.3.0&request=GetCapabilities) actually more that listed. That way one can ask for the projection you want, like this [EPSG:32633](https://thredds.t.niva.no/thredds/wms/datasets/exceedance-limits/fab-sswc-veg-exceedence/2017-2021_acdd_fab-sswc-veg-exceedence.nc?FORMAT=image/png&TRANSPARENT=TRUE&STYLES=default-scalar/psu-viridis&LAYERS=fab&COLORSCALERANGE=0,50&NUMCOLORBANDS=20&ABOVEMAXCOLOR=extend&BELOWMINCOLOR=extend&BGCOLOR=extend&LOGSCALE=false&SERVICE=WMS&VERSION=1.1.1&REQUEST=GetMap&SRS=EPSG:32633&BBOX=-88924.71956845932,6431522.316937,1120220.0,8015886.489092&WIDTH=1024&HEIGHT=1024) request or this [EPSG:4326](https://thredds.t.niva.no/thredds/wms/datasets/exceedance-limits/fab-sswc-veg-exceedence/2017-2021_acdd_fab-sswc-veg-exceedence.nc?FORMAT=image/png&TRANSPARENT=TRUE&STYLES=default-scalar/psu-viridis&LAYERS=fab&COLORSCALERANGE=0,50&NUMCOLORBANDS=20&ABOVEMAXCOLOR=extend&BELOWMINCOLOR=extend&BGCOLOR=extend&LOGSCALE=false&SERVICE=WMS&VERSION=1.1.1&REQUEST=GetMap&SRS=EPSG:4326&BBOX=3.9670144291538,57.173611543323,16.231482057764,69.438079171933&WIDTH=1024&HEIGHT=1024) request. But it is probably best to use a client like qgis for building the requests:)

IvanaH8 commented:
Thank you @kariaust, we have now removed the model keyword.

The title in the map is the same as in the notebook that we changed according to suggestions from your email:
title="Test dataset exceedances of critical loads for acidification and eutrophication 2017-2021",
title_no="Test dataset overskridelser av tålegrenser for forsuring og overgjødsling 2017-2021",

Is something missing/incorrect?

kariaust commented:
The title is fine now (it has changed since I last checked)
